stories/Grid.story.jsx

```diff
@@ -1,33 +1,312 @@
-        <div className="example">
-          <h4>Row</h4>
-          <Row className="example-grid">
-            <div className="example-box">Box 1</div>
-            <div className="example-box">Box 2</div>
-            <div className="example-box">Box 3</div>
-          </Row>
-        </div>
-        <div className="example">
-          <h4>Column</h4>
-          <Col>
-            <div className="example-box">Box 1</div>
-            <div className="example-box">Box 2</div>
-            <div className="example-box">Box 3</div>
-          </Col>
-        </div>
-      </FlexGrid>
-    </div>
-  </ExampleStyles>
-));
+const BoxYellow = styled.div`
+  background-color: #ffff19;
+  height: 50px;
+`;
+
+storiesOf("FlexGrid", module)
+  .addWithJSX(
+    "default",
+    withInfo()(() => (
+      <FlexRow>
+        <FlexColumn col={3}>
+          <Box />
+        </FlexColumn>
+        <FlexColumn col={3}>
+          <Box />
+        </FlexColumn>
+        <FlexColumn col={3}>
+          <Box />
+        </FlexColumn>
+      </FlexRow>
+    ))
+  )
+  .addWithJSX(
+    "around",
+    withInfo()(() => (
+      <FlexRow space="around">
+        <FlexColumn col={3}>
+          <Box />
+        </FlexColumn>
+        <FlexColumn col={3}>
+          <Box />
+        </FlexColumn>
+        <FlexColumn col={3}>
+          <Box />
+        </FlexColumn>
+      </FlexRow>
+    ))
+  )
+  .addWithJSX(
+    "evenly",
+    withInfo()(() => (
+      <FlexRow space="evenly">
+        <FlexColumn col={3}>
+          <Box />
+        </FlexColumn>
+        <FlexColumn col={3}>
+          <Box />
+        </FlexColumn>
+        <FlexColumn col={3}>
+          <Box />
+        </FlexColumn>
+      </FlexRow>
+    ))
+  )
+  .addWithJSX(
+    "between",
+    withInfo()(() => (
+      <FlexRow space="between">
+        <FlexColumn col={3}>
+          <Box />
+        </FlexColumn>
+        <FlexColumn col={3}>
+          <Box />
+        </FlexColumn>
+        <FlexColumn col={3}>
+          <Box />
+        </FlexColumn>
+      </FlexRow>
+    ))
+  )
+  .addWithJSX(
+    "start",
+    withInfo()(() => (
+      <FlexRow position="start">
+        <FlexColumn col={3}>
+          <Box />
+        </FlexColumn>
+        <FlexColumn col={3}>
+          <Box />
+        </FlexColumn>
+      </FlexRow>
+    ))
+  )
+  .addWithJSX(
+    "center",
+    withInfo()(() => (
+      <FlexRow position="center">
+        <FlexColumn col={3}>
+          <Box />
+        </FlexColumn>
+        <FlexColumn col={3}>
+          <Box />
+        </FlexColumn>
+      </FlexRow>
+    ))
+  )
+  .addWithJSX(
+    "end",
+    withInfo()(() => (
+      <FlexRow position="end">
+        <FlexColumn col={3}>
+          <Box />
+        </FlexColumn>
+        <FlexColumn col={3}>
+          <Box />
+        </FlexColumn>
+      </FlexRow>
+    ))
+  )
+  .addWithJSX(
+    "offset",
+    withInfo()(() => (
+      <div>
+        <FlexRow space="around" style={{ marginBottom: 10 }}>
```

Comment:
Can we not repeat this style? For example, should we have a container grid class that allows us to apply styles to all the columns and rows so we don't have to repeat ourselves.